tests/components/recorder/test_init.py

```diff
@@ -361,7 +361,7 @@ def test_auto_purge(hass_recorder):
     dt_util.set_default_time_zone(tz)
 
     now = dt_util.utcnow()
-    test_time = tz.localize(datetime(now.year + 1, 1, 1, 4, 12, 0))
+    test_time = now + timedelta(days=365)
```

Comment:
What is the idea with the time for the test? The docstring seems stale and doesn't explain it.

Reply:
I rewrote the test to be more clear and added a comment.  The original fix was done in a hurry to get tests passing again, so I kind of hacked it together quickly.

Reply:
Nice!